src/goto-symex/witnesses.h

```diff
@@ -33,7 +33,7 @@ class nodet
 class edget
 {
 private:
-  static short int _id;
+  static unsigned int _id;
```

Comment:
why is this static?

Reply:
I think it is because we call `new nodet()` and `new edget()` multiple times in `witnesses.cpp`. We want only one copy of the static member shared by all objects, thereby accumulating the counter `_id += 1` in the constructor.